css-aam/index.html

```diff
@@ -0,0 +1,169 @@
+<!doctype html>
+<html lang="en-US">
+    <head>
+        <meta charset="utf-8" />
+        <title>CSS Accessibility API Mappings 1.0</title>
+        <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove" defer="defer"></script>
+        <script src="../common/script/resolveReferences.js" class="remove"></script>
+        <script src="../common/script/utility.js" class="remove"></script>
+        <script src="../common/script/participants.js" class="remove"></script>
+
+        <script class="remove">
+            var respecConfig = {
+                group: "aria",
+                github: "w3c/aria/css-aam/",
+                doJsonLd: true,
+
+                // Document info
+                specStatus: "ED",
+                shortName: "css-aam-1.0",
+                //publishDate: "2009-08-06",
+                //previousMaturity: "WD",
+                //previousPublishDate: "2012-10-25",
+                //previousURI: "http://www.w3.org/TR/",
+                copyrightStart: "2016",
+                edDraftURI: "https://w3c.github.io/css-aam/",
+                // lcEnd:  "2010-08-06",
+
+                tocIntroductory: true,
+                maxTocLevel: 2,
+
+                // Editors
+                editors: [
+                    { name: "Zoë Bijl", company: "Invited Expert", mailto: "editor@moiety.me", w3cid: 74040 },
+                    { name: "James Teh", company: "Mozilla", companyURL: "https://mozilla.org/" },
```

Comment:
Strictly speaking, I work for Mozilla Corporation, but our W3C membership is under Mozilla Foundation, so...

```suggestion
                    { name: "James Teh", company: "Mozilla Foundation", companyURL: "https://mozilla.org/", mailto: "jteh@mozilla.com", w3cid: 134488 },
```